recipes/ngless/meta.yaml

```diff
@@ -1,80 +1,44 @@
 {% set name = "ngless" %}
-{% set version = "1.3.0" %}
-{% set sha256 = "b0a114ed901b378189b09d091308e5ad554fdbbd7d3722c935d26c0ddd7328d6" %}
-{% set osx_binaries_sha256 = "e72fa4bc5d1666432d4a86c417ec7f80fa21cb4c1f4b3ba5c6490fe8c7f140f9" %}
+{% set version = "1.4.0" %}
+{% set linux_sha256 = "f83f727965c1230eaac0bc1c8a6c6951293da99b589862ba8c5d8a0ff4dbddd3" %}
+{% set osx_sha256 = "a2195c811358b891c84392ebe5f21167b6efbdeaa086891ab4c35f4667d858bd" %}
 
-{% set ghc_src_version = "8.6.5" %}
-{% set ghc_src_sha256 = "4d4aa1e96f4001b934ac6193ab09af5d6172f41f5a5d39d8e43393b9aafee361" %}
-{% set ghc_bootstrap_version = "8.4.2" %}
-{% set ghc_bootstrap_linux_sha256 = "28c31489ed9a83af4400685ec8bc72a6d43d08958d75b2dc35d29a894a5c9d93" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  - url: https://github.com/ngless-toolkit/ngless/archive/v{{ version }}.tar.gz # [linux]
-    sha256: {{ sha256 }} # [linux]
-    folder: ngless # [linux]
-  - url: https://downloads.haskell.org/~ghc/{{ ghc_bootstrap_version }}/ghc-{{ ghc_bootstrap_version }}-x86_64-centos67-linux.tar.xz  # [linux]
-    sha256: {{ ghc_bootstrap_linux_sha256 }}  # [linux]
-    folder: ghc_bootstrap # [linux]
-
-  - url: https://downloads.haskell.org/~ghc/{{ ghc_src_version }}/ghc-{{ ghc_src_version }}-src.tar.xz # [linux]
-    sha256: {{ ghc_src_sha256 }} # [linux]
-    folder: ghc_src # [linux]
-
-  - url: https://github.com/ngless-toolkit/ngless/releases/download/v{{ version }}/NGLess-{{ version }}-osx.zip # [osx]
-    sha256: {{ osx_binaries_sha256 }} # [osx]
-
+  - url: https://github.com/ngless-toolkit/ngless/releases/download/v{{ version }}/NGLess-v{{ version }}-Linux-static-no-deps # [linux]
+    sha256: {{ linux_sha256 }} # [linux]
+  - url: https://github.com/ngless-toolkit/ngless/releases/download/v{{ version }}/NGLess-v{{ version }}-MacOSX.zip # [osx]
+    sha256: {{ osx_sha256 }} # [osx]
 
 build:
-  number: 1
-
+  number: 0
 
 requirements:
-  build:
-    - make
-    - {{ compiler('c') }} # [linux]
-    - {{ compiler('cxx') }} # [linux]
-    - make # [linux]
-    - perl # [linux]
-    - coreutils # [linux]
-    - binutils # [linux]
-    - pkg-config # [linux]
-    - automake # [linux]
-    - autoconf # [linux]
-    - libtool # [linux]
-    - stack >=1.9.3 # [linux]
-  host:
-    - gmp
-    - xz
-    - zlib
-    - bzip2
-    - backports.lzma
-    - libiconv
-    - openssh
-    - wget # [linux]
-    - libffi # [linux]
   run:
-    - libffi # [linux]
-
+    - bwa
+    - prodigal
+    - megahit
+    - samtools
+    - minimap2
```

Comment:
Is version pinning needed in any of the above given they are distributed through conda?

Reply:
Otherwise, the rest LGTM

Reply:
I decided against version pinning as fitting more with the conda model

These are (currently) stable enough packages (except minimap2, which Ngless does _not_ default to), none of them have had updates that change results in the last few years.